config/abilities/master_member.rb

```diff
@@ -16,6 +16,7 @@
       end
     end
 
+    can(:create, Account, &:signup_provider_possible?)
```

Comment:
What the controller is doing is:
https://github.com/3scale/porta/blob/c865ad5db23bd45c12b2d6ee57b39eafcbb01890/app/controllers/provider/admin/accounts_controller.rb#L10
https://github.com/3scale/porta/blob/c865ad5db23bd45c12b2d6ee57b39eafcbb01890/app/controllers/provider/admin/accounts_controller.rb#L65-L67
https://github.com/3scale/porta/blob/c865ad5db23bd45c12b2d6ee57b39eafcbb01890/app/lib/logic/provider_signup.rb#L9-L12

Imo in the case of member users we should check also `user.has_permission?(:partners)` but that would be changing the current behaviour so that's why I didn't do it 🤔